lnst/External/TRex/UDPSimple.py

```diff
@@ -10,6 +11,18 @@ class UDPSimple(object):
     port_id:  The port the stream will be added to
     """
 
+    def __init__(self):
+        self._import_optionals()
+
+    @staticmethod
+    def _import_optionals():
+        try:
+            from trex_stl_lib.api import *
+        except ModuleNotFoundError:
+            msg = f"Module trex_stl_lib not found, please install it"
```

Comment:
Same comment as for https://github.com/LNST-project/lnst/pull/244/commits/a4d36b165adc60b8bfa783e70823b3a9172c1abb#r886815852